.github/workflows/CI.yml

```diff
@@ -225,6 +225,33 @@ jobs:
         if: matrix.version != 'nightly' || steps.run_tests.outcome == 'success'
         with:
           files: lcov.info
+  integration:
+    name: Integration Tests - ${{ matrix.test }}
+    runs-on: ${{ matrix.os }}
+    env:
+      JULIA_PKG_SERVER_REGISTRY_PREFERENCE: eager
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
```

Comment:
Minor but can you change to 1.10 explicitly for now [will bump once we have 1.11 support]

Reply:
Will do